README.md

```diff
@@ -298,6 +297,11 @@ terms, and open source software license terms. These separate license terms
 govern your use of the third party programs as set forth in the
 "[THIRD-PARTY-PROGRAMS](THIRD-PARTY-PROGRAMS)" file.
 
---------
+# Legal Information
 
-[Legal Information](doc/legal_information.md)
```

Comment:
Let me see what content is actually needed here. Meanwhile please revert the change.

Reply:
Done